rs/sns/cli/src/lib.rs

```diff
@@ -57,6 +59,19 @@ const TEST_NEURON_1_OWNER_DFX_IDENTITY_NAME: &str =
 pub struct CliArgs {
     #[clap(subcommand)]
     pub sub_command: SubCommand,
+
+    /// Override the compute network to connect to. By default, the local network is used.
+    /// A valid URL (starting with `http:` or `https:`) can be used here, and a special ephemeral
+    /// network will be created specifically for this request. E.g. "http://localhost:12345/"
+    /// is a valid network name.
```

Comment:
Is it actually true given the code in this PR that a special ephemeral network will be created specifically for this request for any valid url? That will be true once #2927 is merged and used here, I didn't realize this was the default behavior of ic-agent.

```suggestion
    /// A valid URL (starting with `http:` or `https:`) can be used here.
```

Reply:
Good catch, thanks! Just copied the comment from DFX's help message